CMakeLists.txt

```diff
@@ -80,3 +80,5 @@ protocol("unstable/linux-dmabuf/linux-dmabuf-unstable-v1.xml" "linux-dmabuf-unst
 
 # Installation
 install(TARGETS hyprlock)
+
+install(FILES "pam/hyprlock" DESTINATION "/etc/pam.d")
```

Comment:
```suggestion
install(FILES "pam/hyprlock" DESTINATION "${CMAKE_INSTALL_SYSCONFDIR}/pam.d")
```

As per https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html.

Reply:
@fufexan using that just makes it install to `/pam.d/hyprlock`
does cmake always need to be called with `-DCMAKE_INSTALL_SYSCONFDIR=/etc`?

Reply:
Oops my bad, forgot we have to also add `include(GNUInstallDirs)` before the`install(...)`.

Reply:
Yes that worked